vscode/microsoft-kiota/README.md

```diff
@@ -1,58 +1,112 @@
 
-<img width="189" alt="Kiota OpenAPI explorer" src="https://user-images.githubusercontent.com/5781590/229947806-27ff49b9-5877-41a2-b7df-c9c19f6f736e.png">
+<img width="482" alt="Use the add file icon" src="https://github.com/microsoft/kiota/blob/865b5f9a3b6aa495f42a397714b09a08bdb99011/vscode/microsoft-kiota/images/samples/SearchOrBrowseOptions.png">
 
-* Select the endpoints to include in your API client 
+The notification bar will also notify you of ongoing background processes e.g when searching for an api using a keyword
 
-<img width="225" alt="kiota vscode select endpoint" src="https://user-images.githubusercontent.com/5781590/229948168-efecfd85-214a-4d65-a225-10b100b15a68.png">
+<img width="482" alt="vscode extension search notification" src="https://github.com/microsoft/kiota/blob/865b5f9a3b6aa495f42a397714b09a08bdb99011/vscode/microsoft-kiota/images/samples/searchingNotification.png">
 
-* Finally, you can generate the API client. You will be prompted to provide some parameters for your client such as the class and namespace names. You will also need to select the language for the generated client.
+<img width="482" alt="vscode extension search results" src="https://github.com/microsoft/kiota/blob/865b5f9a3b6aa495f42a397714b09a08bdb99011/vscode/microsoft-kiota/images/samples/searchResults.png">
 
-<img width="349" alt="generation complete" src="https://user-images.githubusercontent.com/5781590/229949052-159f3a58-b0e6-421f-9ca5-b45dc49c4639.png">
+Select the OpenAPI description you are interested in and you will be presented with the Kiota OpenAPI Explorer containing all the available endpoints 
 
+<img width="482" alt="Kiota OpenAPI explorer" src="https://github.com/microsoft/kiota/blob/865b5f9a3b6aa495f42a397714b09a08bdb99011/vscode/microsoft-kiota/images/samples/endpointSelectionandTheGenerateIcon.png">
 
-## Requirements
+Select the endpoints to include in your API client as above and click the `generate` icon. Kiota extension will display with the options to generate either client, plugin or other.
+
+<img width="482" alt="kiota vscode generate options" src="https://github.com/microsoft/kiota/blob/865b5f9a3b6aa495f42a397714b09a08bdb99011/vscode/microsoft-kiota/images/samples/SelectGenerationOption.png">
+
+Finally, the notification bar at the bottom right will display "Generation Completed Successfully" once the generation is done. Click on your vscode file explorer to see the generated outputs in the current workspace if default selection was your output directory choice or navigate to selected folder to see the output.
+
+## Migrating from older use of lock file to a workspace.json
+
+Latest Kiota extension has been improved to using a workspace instead of a lock file to facilitate having multiple clients, plugins, or manifests within the same workspace. The workspace.json file also provides an opportunity to edit, or regenerate the outputs conveniently.
```

Comment:
```suggestion
The latest version of the Kiota extension has been improved to using a workspace instead of a lock file to facilitate having multiple clients, plugins, or manifests within the same workspace. The workspace.json file also provides an opportunity to edit, or regenerate the outputs conveniently.
```